aspnetcore/blazor/class-libraries.md

```diff
@@ -20,45 +20,72 @@ Components can be shared in Razor class libraries across projects. Components ca
 
 Just as components are regular .NET types, components provided by Razor class libraries are normal .NET assemblies.
 
-Use the `razorclasslib` (Razor class library) template with the [dotnet new](/dotnet/core/tools/dotnet-new) command:
+## Create a Razor class library
 
-```console
-dotnet new razorclasslib -o MyComponentLib1
-```
+# [Visual Studio](#tab/visual-studio)
+
+To create a Razor class library in Visual Studio:
+
+1. Create a new project.
+1. Select **ASP.NET Core Web Application**. Select **Next**.
+1. Provide a project name in the **Project name** field or accept the default project name. The examples in this topic use the project name `MyComponentLib1`. Confirm the **Location** entry is correct or provide a location for the project. Select **Create**.
+1. In the **Create a new ASP.NET Core Web Application** dialog, confirm that **.NET Core** and **ASP.NET Core 3.0** are selected.
```

Comment:
Can't you just say"

confirm that ~**.NET Core** and~ **ASP.NET Core 3.0** is selected.

Reply:
Don't think so ... the first dropdown generates the second. If we only say the second, it's not crystal clear where "ASP.NET Core 3.0" comes from. Safest is to say both.